tests/conditional_compilation/conftest.py

```diff
@@ -48,6 +48,12 @@ def pytest_addoption(parser):
         type=Path,
         help="Path to the benchmark_app tool",
     )
+    parser.addoption(
+        "--collector_dir",
```

Comment:
@asomsiko what is the reason to specify benchmark_app root dir explicitly? I think we just need to specify full path to a benchmark_app, and, if required, specify `--bindir=benchmark_app.parent`

Reply:
@vurusovs, the collector is a libIntelSEAPI.so and its location on Linux differ from benchmark_app.parent

Reply:
@asomsiko You are right, I misunderstood the necessity of the argument